osquery/logger/logger.cpp

```diff
@@ -408,10 +372,6 @@ void BufferedLogSink::send(google::LogSeverity severity,
                            const struct ::tm* tm_time,
                            const char* message,
                            size_t message_len) {
-  if (FLAGS_disable_logging) {
```

Comment:
This check is not necessary as BufferedLogSink is registered (its setUp is called) only when this flag is true.

Reply:
send cannot be called until BufferedLogSink in GLOG